src/pages/Main/Proudct.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import './Main.scss';
+import SubProduct from './SubProudct';
+
+const Product = props => {
+  const { item } = props;
+
```

Comment:
```suggestion
const Product = {props} => {
  const { title, text,  } = item;
```
이렇게 구조분해 할당을 한다면 조금 더 편리하게 사용할 수 있을거에요!